<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>530</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CHAPÉU UNISSEX ONE PIECE - BRANCO&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 30,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>201</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/bones/One piece, Naruto, Simpsons/onepiece-branco-bone.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/bones/One piece, Naruto, Simpsons/simpsons-preto-bone.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>530</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CHAPÉU UNISSEX NARUTO - BRANCO&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 30,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/bones/One piece, Naruto, Simpsons/naruto-preto-bone.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/bones/One piece, Naruto, Simpsons/naruto-branco-bone.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_chapeu_branco_simpsons">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>580</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>530</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CHAPÉU UNISSEX SIMPSONS - BRANCO&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 30,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>6</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;CHAPÉUS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>201</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/bones/One piece, Naruto, Simpsons/onepiece-preto-bone.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>360</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/bones/One piece, Naruto, Simpsons/simpsons-branco-bone.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icons/icones/Gamepad(1).png);
image: url(:/icons/icones/Gamepad (3).png);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_chapeu_branco_naruto">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>580</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_chapeu_branco_onep">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>580</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CHAPÉU UNISSEX ONE PIECE - PRETO&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 30,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_chapeu_preto_onep">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_chapeu_preto_simpsons">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>290</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>240</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CHAPÉU UNISSEX SIMPSONS - PRETO&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 30,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_chapeu_preto_naruto">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CHAPÉU UNISSEX NARUTO - PRETO&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 30,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_voltar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>VOLTAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_carrinho">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/imgicons/icones/carr.png);

color: rgb(255, 255, 255);
border-style:outset;
border-width: 2px;
border-color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img-icons.qrc"/>
  <include location="../produtos/produtos.qrc"/>
 </resources>
 <connections/>
</ui>
